api/v1beta1/placementapi_types.go

```diff
@@ -46,9 +46,9 @@ type PlacementAPISpec struct {
 	ServiceUser string `json:"serviceUser"`
 
 	// +kubebuilder:validation:Required
-	// MariaDB instance name
-	// Right now required by the maridb-operator to get the credentials from the instance to create the DB
-	// Might not be required in future
+	// MariaDB instance name.
+	// This can't be updated unless advanced mode is enabled, because updating this does not migrate
+	// data to a new database instance.
```

Comment:
Will updating this to a new database instance at least ensures that the DB schema is applied to the new DB instance?